cmd/nexd/main.go

```diff
@@ -73,6 +74,11 @@ func nexdRun(cCtx *cli.Context, logger *zap.Logger, mode nexdMode) error {
 		logger.Info("Starting in L4 proxy mode")
 	}
 
+	stunServers := cCtx.StringSlice("stun-server")
+	if stunServers != nil {
+		stun.SetServers(stunServers)
+	}
```

Comment:
How about validating that there are at least 2 since we expect at least 2 for symmetric NAT discovery?  They also need to be different.

Reply:
Well, in the case of e2e tests we only set 1 stun server.

Reply:
how about spinning up 2 on 2 different ports for e2e?